doit/task.py

```diff
@@ -189,6 +191,7 @@ def __init__(self, name, actions, file_dep=(), targets=(),
         self.check_attr(name, 'getargs', getargs, self.valid_attr['getargs'])
         self.check_attr(name, 'title', title, self.valid_attr['title'])
         self.check_attr(name, 'watch', watch, self.valid_attr['watch'])
+        self.check_attr(name, 'cfg_defaults', cfg_defaults, self.valid_attr['cfg_defaults'])
```

Comment:
All these initializations and check are for values provided by users in `dodo.py` files when creating tasks.
That is not the case for these config values. So no need to change `Task.__init__`, just initialize the value to `None`.

Reply:
Fixed. I've removed the initializations and checks.